src/main/java/wedlog/address/logic/commands/VendorDeleteCommand.java

```diff
@@ -1,13 +1,20 @@
 package wedlog.address.logic.commands;
 
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
 import wedlog.address.commons.core.index.Index;
+import wedlog.address.commons.util.ToStringBuilder;
+import wedlog.address.logic.Messages;
 import wedlog.address.logic.commands.exceptions.CommandException;
 import wedlog.address.model.Model;
+import wedlog.address.model.person.Vendor;
 
 /**
  * Deletes a Vendor from the address book.
  */
-public class VendorDeleteCommand extends Command { // this is still the old DeleteCommand Implementation
+public class VendorDeleteCommand extends Command {
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
```

Comment:
Could you add in the "vendor" keyword as well? We can create it as a public static final under `VendorCommandParser` class.